src/content/en/tools/workbox/guides/_toc.yaml

```diff
@@ -44,10 +46,9 @@ toc:
       path: /web/tools/workbox/guides/codelabs/gulp
     - title: "Workbox For npm Script"
       path: /web/tools/workbox/guides/codelabs/npm-script
-    # Needs updating to v3
-    # - title: "Build a PWA using Workbox"
-    #   path: "https://codelabs.developers.google.com/codelabs/workbox-lab/"
-    #   status: external
+    - title: "Build a PWA using Workbox"
```

Comment:
While updating the ToC, I realized that we still had this commented out, even though the codelab has been updated for v3.